x/observer/keeper/grpc_query_ballot.go

```diff
@@ -93,3 +93,20 @@ func (k Keeper) Ballots(goCtx context.Context, req *types.QueryBallotsRequest) (
 	}
 	return &types.QueryBallotsResponse{Ballots: ballots, Pagination: pageRes}, nil
 }
+
+func (k Keeper) BallotListForHeight(
```

Comment:
What is this query necessary? Debugging purpose?

Still to add lot of LoC without clear reason